client/components/register/imports/RegisterForm.jsx

```diff
@@ -225,7 +225,7 @@ class RegisterForm extends Component {
         <Form.Group widths='equal'>
           <Form.Input name='zip' label='Zip/Postal Code' placeholder='e.g. 98225' value={ this.state.zip }
                       onChange={ (e) => this._handleTextChange(e) }/>
-          <Form.Dropdown name='state' label='State/Province' search selection options={ STATES }
+          <Form.Dropdown name='state' label='State/Province (or other)' search selection options={ STATES }
```

Comment:
nit: I think it might be clearer if "other" was in quotes (ie `State/Province (or "other")` for those registering from outside the country

Reply:
Made that change.